app/src/main/java/com/revieve/PluginWebViewJavaSample/FullscreenActivity.java

```diff
@@ -212,11 +216,34 @@ public void onProgressChanged(WebView view, int progress) {
         webSettings.setGeolocationEnabled(true);
         webSettings.setAllowContentAccess(true);
         webSettings.setMediaPlaybackRequiresUserGesture(false);
+        myWebView.setWebContentsDebuggingEnabled(true);
         myWebView.addJavascriptInterface(new RevieveJSInterface(this), "Android");
 
         myWebView.loadUrl(REVIEVE_FULL_URL);
     }
 
+    @Override
+    protected void onPause () {
+        super.onPause();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            myWebView.postWebMessage(
+                    new WebMessage("{\"type\": \"setCameraPause\", \"payload\": true}"),
+                    Uri.parse(REVIEVE_CDN_DOMAIN)
+            );
+        }
+    }
+
+    @Override
+    protected void onResume () {
+        super.onResume();
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+            myWebView.postWebMessage(
+                    new WebMessage("{\"type\": \"setCameraPause\", \"payload\": false}"),
+                    Uri.parse(REVIEVE_CDN_DOMAIN)
+            );
+        }
+    }
+
```

Comment:
I think it would be great to comment in the code why they might want to use this snippet and also mention webview.pause / resume alternatives